test/Microsoft.AspNet.Mvc.FunctionalTests/TagHelperSampleTest.cs

```diff
@@ -8,6 +8,8 @@
 using System.Threading.Tasks;
 using Microsoft.AspNet.Builder;
 using Microsoft.AspNet.TestHost;
+using Microsoft.Framework.DependencyInjection;
```

Comment:
If that's true then we shouldn't have the test at all. We have a site called `TagHelperSample.Web`, so I put Tag Helper stuff in it and in this case it required some new dependencies. I don't see it happening very often.